kibbeh/src/stories/LeftHeader.story.tsx

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck @todo this file needs to be fixed
```

Comment:
you can remove this, I put this in just to get things to compile

Reply:
These have been removed from all stories in this commit